src/qt/receivecoinsdialog.cpp

```diff
@@ -41,6 +43,17 @@ ReceiveCoinsDialog::ReceiveCoinsDialog(const PlatformStyle *_platformStyle, QWid
         ui->removeRequestButton->setIcon(_platformStyle->SingleColorIcon(":/icons/remove"));
     }
 
+    // configure bech32 checkbox, disable if launched with legacy as default:
+    if (model->getDefaultAddressType() == OUTPUT_TYPE_BECH32) {
+      ui->useBech32->setCheckState(Qt::Checked);
```

Comment:
nit: 2 spaces rather then 4 spaces indentation? Not sure if that is after the guidelines...

Reply:
It's not, it's my editor misbehaving. Fixed (in three places).